src/views/IndividualTeamView.vue

```diff
@@ -6,16 +6,20 @@
     <div v-if="!error && loaded">
       <h1>
         {{ team.name }}
-        <!-- <img v-if="permissionLevel == 2" src="../assets/edit-icon.svg" alt="edit"> -->
-        <img v-if="permissionLevel == constants.NONE"
+        <!-- <img v-if="userTeamRole == teamConstants.LEADER"
```

Comment:
Is there any reason to keep these comments here?

Reply:
The figma designs show this functionality, left the comments as a reminder to do it but they're easy to add back in if you want them removed